micro_sam/evaluation/evaluation.py

```diff
@@ -75,4 +74,55 @@ def run_evaluation(
     return results
 
 
+def run_evaluation_for_interactive_prompting(
```

Comment:
Let's rename this to `run_evaluation_for_iterative_prompting`. We call this `iterative_prompting` in all other functions, so it would be confusing to change the naming convention only here.

Reply:
I am not sure if you're using this anywhere else in this PR, if yes you of course also need to adapt the name there.